packages/db/prisma/migrations/20230721002048_add_blob_data_and_data_reference_models/migration.sql

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
At some point we will have to include also migration scripts for production. But so far with devnet is fine